src/analysis/special_functions/integrals.lean

```diff
@@ -20,6 +20,26 @@ This file is incomplete; we are working on expanding it.
 open real set interval_integral
 variables {a b : ℝ}
 
+namespace interval_integral
+variable {f : ℝ → ℝ}
+
+@[simp]
+lemma integral_const_mul (c : ℝ) : ∫ x in a..b, c * f x = c * ∫ x in a..b, f x :=
+integral_smul c
+
+@[simp]
+lemma integral_mul_const (c : ℝ) : ∫ x in a..b, f x * c = (∫ x in a..b, f x) * c :=
+by simp only [mul_comm, integral_const_mul]
+
+@[simp]
+lemma integral_div' (c : ℝ) : ∫ x in a..b, f x / c = (∫ x in a..b, f x) * c⁻¹ :=
```

Comment:
I believe `simp` doesn't choose between `x / y` or `x * y⁻¹` as simp-normal form, so it seems unnatural that this simp lemma changes the former form into the latter form...

Reply:
Let me see if I can add `simp` to `integral_div` instead. If that works I should be able to remove this lemma entirely and anything involving `inv` should be true by `simp` all the same.

Reply:
@fpvandoorn Done!